encoding/preparation.md

```diff
@@ -250,7 +251,85 @@ such as Intel's [SVT-AV1][] will also not be included.
 
 ## Audio Codecs
 
-TODO: FLAC, AAC, maybe mention opus
+### Audio formats and how to handle them
+
+Depending on the source you'll be working with,
+you may encounter many different audio formats.
+
+On Blu-rays,
+you will most likely find audio encoded losslessly,
+in the form of either DTS-HD Master Audio, Dolby TrueHD, or PCM.
+DTS-HD MA and Dolby THD are proprietary codecs that use lossless compression,
+while PCM is simply raw, uncompressed PCM data.
+The usual way to handle these formats is to reencode them to other formats -
+either lossless or lossy, depending on your taste.  
+But first, you need to decode them.
+The recommended tool for that is FFmpeg.
+You can find Windows builds and Linux packages
+on [FFmpeg's official site][ffmpeg].
+
+When working with WEB and TV sources,
+you will most likely have only lossy audio available.
+The most common codecs here are AC-3, E-AC-3 and AAC.
+Lossily compressed files should generally not be reencoded -
+the proper way to handle them is to remux (i.e. copy) them to the final file.
+
+[ffmpeg]: https://www.ffmpeg.org/download.html
+
+### Which codecs to use?
+
+Once you have your lossless files decoded, you need to encode them.
+Depending on your taste, you can choose a lossy or lossless codec.
+The two most widely accepted codecs in fansubbing community are FLAC (lossless)
+and AAC (lossy), but recently opus (also lossy) is gaining some popularity, too.
+
+The recommended encoder for FLAC is the official one.
+Download Windows builds from [it's official site][FLAC].
+Most Linux distributions should have FLAC in their package repositories.
+
+The recommended and most widely used AAC encoder is qaac,
+available on [it's official site][qaac]. 
+Nero and Fraunhofer FDK aren't really that much worse,
+so you can use them if you really want.
+Other AAC encoders are discouraged,
+since they provide inferior results.
+
+There is also opus, which is gaining some popularity recently.
+It is currently the most efficient lossy codec,
+and it's completely FOSS if you're into that.
+The recommended opus encoder is the official one,
+contained in the [opus-tools package][opus].
+
+Other codecs are generally not recommended.
+Formats like Monkey's Audio and TAK provide very little gain over FLAC,
+while not being as widely supported,
+and — in the case of TAK —
+closed source.
+DTS-HD MA and Dolby THD are much less efficient than FLAC,
+and are also closed source.
+MP3 is simply obsolete,
+and Vorbis has been superseded by opus.
+DTS and AC-3 provide even worse compression than MP3,
+and don't have any reasonable, free encoders.  
+In short - don't bother, unless you really have to, for some reason.
+
+[FLAC]: https://xiph.org/flac/download.html
+[qaac]: https://sites.google.com/site/qaacpage/cabinet
+[opus]: https://opus-codec.org/downloads/
+
+### Lossless or lossy?
```

Comment:
Section headers/titles not directly under another header should have two blank lines before them, separating them from the previous section.